dapps/digital-twin-detail/src/components/overview/DetailOverview.ts

```diff
@@ -17,13 +17,84 @@
   the following URL: https://evan.network/license/
 */
 
-// vue imports
 import Component, { mixins } from 'vue-class-component';
-
-// evan.network imports
 import { EvanComponent } from '@evan.network/ui-vue-core';
+import {
+  Runtime,
+  Profile,
+  ProfileOptions,
+} from '@evan.network/api-blockchain-core';
+import { DAppTwin, TwinTransaction } from '@evan.network/digital-twin-lib';
+import { bccUtils } from '@evan.network/ui';
+import { CommunicationService } from './CommunicationService';
 
 @Component
-export default class EvanTwinDetailOverviewComponent extends mixins(EvanComponent) {
+export default class DetailOverviewComponent extends mixins(EvanComponent) {
+  twin: DAppTwin = null;
+
+  runtime: Runtime = null;
+
+  transactions: TwinTransaction[] = null;
+
+  communicationService: CommunicationService = null;
+
+  async created(): Promise<void> {
+    this.runtime = this.getRuntime();
+    this.communicationService = new CommunicationService(this.runtime);
+    this.twin = this.$store.state.twin;
```

Comment:
would it make sense to define this as computed value?

something like this? 
```
get twin () {
  return this.$store.state.twin;
}
```

Reply:
Should be okay like this. The twin is set using `$set` within the root component to handle object reactivity of loading flags / update flags.